src/ui/components/core/MessageHistory.tsx

```
@@ -37,11 +39,55 @@ export default function MessageHistory({ messages }: MessageHistoryProps) {
           </Box>
         );
         
-      // TODO: Add symbol in front of each assistant message
       case 'assistant':
+        const markdownElements = parseMarkdown(message.content);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)